### ISSUE TITLE
Fix adding location

### ISSUE THREAD

**Issue by kdiaz489** [kigt-inc/charge-cloud-backend-2023#53]
Not able to add location to a user

![Image](https://github.com/kigt-inc/charge-cloud-backend-2023/assets/26280604/7bba00e4-3ad1-46ff-95f8-fc1ab06ccb49)

**Comment by darshanP-crest** [comment]
@kdiaz489 
This error happen because of the client with id 8 is does not exist on our database.

**Comment by kdiaz489** [comment]
@darshanP-crest i see the user with id 8 here? 



![Image](https://github.com/kigt-inc/charge-cloud-backend-2023/assets/26280604/e43178f3-5b2c-4e4d-894f-e7f17ef6474c)

**Comment by darshanP-crest** [comment]
@kdiaz489 
No I am talking about client not the user!
According to our last discussion client owns the locations right?

**Comment by kdiaz489** [comment]
@darshanP-crest Yes! My apologies i was using the user id. I will close this task

**Comment by darshanP-crest** [comment]
@kdiaz489 It's Ok No problem!
